cirq/ops/eigen_gate.py

```diff
@@ -227,3 +234,45 @@ def _is_parameterized_(self) -> bool:
     def _resolve_parameters_(self: TSelf, param_resolver) -> TSelf:
         return self._with_exponent(
                 exponent=param_resolver.value_of(self._exponent))
+
+
+def _lcm(vals: Iterable[int]) -> int:
+    t = 1
+    for r in vals:
+        t = t * r // fractions.gcd(t, r)
+    return t
+
+
+def _approximate_common_period(periods: List[float],
+                               initial_denom: int = 60,
+                               reject_atol: float = 1e-8) -> Optional[float]:
+    if not periods:
```

Comment:
doc these args.  initial_denom and reject_atol are currently not publicly accessible, I guess they are being used for testing?

Reply:
Done.